src/central_limit_theorem/PopulationForm.jsx

```diff
@@ -61,7 +61,7 @@ export const PopulationForm  = (
                     <div className="form-group">
                         <div className="form-row ">
                             <label htmlFor="distType">
-                                Distribution Type</label>
+                                Choose a Distribution Type</label>
```

Comment:
Like the directive rather than just labels. Do you need a colon here for consistency?